numpy/random/mtrand.pyx

```diff
@@ -3481,6 +3482,99 @@ cdef class RandomState:
         # Match historical output type
         return int64_to_long(out)
 
+    def two_sided_geometric(self, alpha, size=None):
```

Comment:
You must not change mtrand.pyx.  This frozen is effectively frozen, and will only receive changes necessary for it to compile in the future.